Source/Cells/BrickCell.swift

```diff
@@ -106,6 +111,15 @@ open class BaseBrickCell: UICollectionViewCell {
     open override func layoutSubviews() {
         super.layoutSubviews()
         brickBackgroundView?.frame = self.bounds
+
+        if frame.width == requestedWidth {
+            self.layoutIfNeeded() // This layoutIfNeeded is added to make sure that the subviews are laid out correctly
```

Comment:
Does calling layoutIfNeeded(), also call layoutSubviews()? or does the "if needed" mean only if self.bounds has changed?  Just trying to make sure that there isn't a condition that this can get into an infinite loop.